src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,78 @@
+import React from 'react';
+import classNames from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(total / perPage);
+
+  const handleCurrentPage = (page: number) => {
+    if (page < 1 || page > totalPages) {
+      return;
+    }
+
+    onPageChange(page);
+  };
+
+  return (
+    <ul className="pagination">
+      <li className={classNames('page-item', { disabled: currentPage === 1 })}>
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          aria-disabled={`${currentPage === 1 ? 'true' : 'false'}`}
+          onClick={() => handleCurrentPage(currentPage - 1)}
+        >
+          «
+        </a>
+      </li>
+      {Array.from({ length: totalPages }).map((_, i) => {
+        const page = i + 1;
+
+        return (
+          <li
+            className={classNames('page-item', {
+              active: currentPage === page,
+            })}
+            key={i}
+          >
+            <a
+              data-cy="pageLink"
+              className="page-link"
+              href={`#${page}`}
+              onClick={() => handleCurrentPage(page)}
+            >
+              {page}
+            </a>
+          </li>
+        );
+      })}
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === totalPages,
+        })}
+      >
+        <a
+          data-cy="nextLink"
+          className="page-link"
+          href="#next"
```

Comment:
Similar to the previous comment, using `href="#next"` is not ideal. Consider using `href="#"` or a button element to prevent default link behavior.